geometry_msgs/Pose sdh_palm_link_pose
---
bool completed
string msg